sdk/metricsadvisor/azure-ai-metricsadvisor/src/test/resources/session-records/DataFeedWithCredentialsTest.testBlobStorage[1].json

```diff
@@ -4,103 +4,108 @@
     "Uri" : "https://REDACTED.cognitiveservices.azure.com/metricsadvisor/v1.0/dataFeeds",
     "Headers" : {
       "User-Agent" : "azsdk-java-azure-ai-metricsadvisor/1.2.0-beta.1 (11.0.9; Windows 10; 10.0)",
-      "x-ms-client-request-id" : "4dcc1081-0946-4f80-986c-b0f236d4b7f2",
+      "x-ms-client-request-id" : "003a1971-e066-4ac7-8c72-068cd64765ee",
```

Comment:
Why did we need to re-record some tests ?

Reply:
I think earlier, there was a bug with one of the API's. deleteDataSourceCredential was earlier routing to deletDatafeed. The updated recordings are now doing the correct thing.